packages/gatsby-theme-newrelic/src/components/Lightbox.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { animated, useTransition } from 'react-spring';
+import { rgba } from 'polished';
+import { css } from '@emotion/react';
+import Portal from './Portal';
+import useKeyPress from '../hooks/useKeyPress';
+import useScrollFreeze from '../hooks/useScrollFreeze';
+
+const Lightbox = ({ children }) => {
+  const [lightboxOpen, setLightboxOpen] = useState(false);
+  const transitions = useTransition(lightboxOpen, {
+    config: { tension: 220, friction: 22 },
+    from: {
+      opacity: 0,
+      transform: 'scale(0.96)',
+    },
+    enter: { opacity: 1, transform: 'scale(1)' },
+    leave: { opacity: 0, transform: 'scale(0.96)' },
+  });
+  useScrollFreeze(lightboxOpen);
+  useKeyPress('Escape', () => setLightboxOpen(false), {
+    ignoreTextInput: false,
+  });
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => setLightboxOpen(true)}
+        css={css`
```

Comment:
do we want to set the background color to transparent instead of that gray color? and also change cursor to something like a magnifying glass on hover?

![2022-05-27_15-38-09](https://user-images.githubusercontent.com/2952843/170797211-0088c8c4-3525-4e3e-81e3-16725414ec1c.png)

Reply:
oh my goodness yep, i got rid of the button background in the dev tools but not in the code 🙈